metadata-editor/app/controllers/EditsController.scala

```diff
@@ -193,6 +201,16 @@ object EditsController extends Controller with ArgoHelpers {
     edits
   }
 
+  def metadataFromUsageRights(usageRights: UsageRights): Option[ImageMetadata] = {
+    val toImageMetadata: PartialFunction[UsageRights, ImageMetadata] = (ur: UsageRights) => ur match {
+      case u: StaffPhotographer        => ImageMetadata(byline = Some(u.photographer), credit = Some(u.publication))
+      case u: ContractPhotographer     => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
+      case u: CommissionedPhotographer => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
```

Comment:
Illustrations fall in to the weird world of not being _Photgraphers_. Agreed - we're going to have to create a new category for those.

Photographers: 
Contract & Commissioned = "______ for the Guardian"
Staff: "The Guardian"

So I think the above stands, we might just have to make Composer aware of staff photographers.